실패율.js

```diff
@@ -0,0 +1,25 @@
+function solution(N,stages) {
+    const failRate = [];
+    const answer = {};
+    const result = [];
+    // 실패율 계산, aswer 객체에 스테이지를 키값, 실패율을 밸류값으로 저장
+    for (let i = 1; i <= N; i++) {
```

Comment:
```suggestion
    for (let i = 1; i <= N; i += 1) {
```
[no-plusplus 규칙](https://eslint.org/docs/rules/no-plusplus)